docs/flashbots-auction/advanced/rpc-endpoint.mdx

```diff
@@ -869,6 +869,167 @@ If the first address matches the authentication signature, then a response with
 
 If the signature is invalid or does not match the first address, an appropriate error will be returned instead.
 
+### MEV Refund JSON-RPC API Documentation
+
+Below we will document set of endpoints to track MEV Refunds generated by MEV-Share. All of these endpoints do not require signature
+
+### flashbots_getMevRefundsByRecipient
+
+Retrieves all refunds sent to a specific recipient address.
```

Comment:
In each of these individual API entries we should:
* Specify that this API is for MEV-Share refunds. Eg. change the text to say "Retrieves all **MEV-Share** refunds..." where "MEV-Share" could also be a link to https://docs.flashbots.net/flashbots-protect/mev-share
* Specify that the API does not require authentication (I would say "authentication" and not "signature")

(This same comment applies to each new API section below).